events/factories.py

```diff
@@ -0,0 +1,33 @@
+import factory
+from django.utils import timezone
+
+from fair.factories import FairFactory
+from .models import Event
+
```

Comment:
What is the logic of structuring the imports like this and/or do we have a preferred way of structuring imports?